bigchaindb/consensus.py

```diff
@@ -89,6 +89,18 @@ def verify_signature(signed_transaction):
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def verify_vote_signature(signed_vote):
+        """Verify a cast vote.
+
+        Args:
+            signed_vote (dict): signed vote to verify
+
+        Returns:
+            bool: True if the votes's required signature data is present
+                and correct, False otherwise.
+        """
+        raise NotImplementedError
```

Comment:
I guess the abstractmethods don't require `NotImplementedError` when using `ABC`.

same goes for the other methods in this abstract class